registry/index.md

```diff
@@ -13,6 +13,8 @@ title: Docker Registry
 The Registry is a stateless, highly scalable server side application that stores
 and lets you distribute Docker images. The Registry is open-source, under the
 permissive [Apache license](https://en.wikipedia.org/wiki/Apache_License).
+You can find the source code on
+[GitHub](https://github.com/distribution/distribution){: target="blank" rel="noopener" class=""}.
```

Comment:
I would have a preference for a more specific mention, such as:
```suggestion
You can find the source code on the
[Distribution GitHub repository](https://github.com/distribution/distribution){: target="blank" rel="noopener" class=""}.
```
Otherwise, looks good to me.